### ISSUE TITLE
pom.xml:117-120: Migrate existing tests to JUnit 5. It...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/cactoos#1416]
The puzzle `1411-445dea43` from #1411 has to be resolved: 

https://github.com/yegor256/cactoos/blob/033b34e1378d4565a95717356bc68dd6ee408063/pom.xml#L117-L120

The puzzle was created by Victor Noël on 05-Jul-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@victornoel[/z](https://www.0crat.com/u/victornoel) please, pay attention to this issue

**Comment by 0crat** [comment]
The job [#1416](https://github.com/yegor256/cactoos/issues/1416) assigned to @borysfan[/z](https://www.0crat.com/u/borysfan), here is [why](https://www.0crat.com/footprint/C63314D6Z/3bc73256-f32b-4b19-83a1-a8327e7c6a57); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/cactoos/issues/1416#issuecomment-658882289))

@victornoel The user @borysfan[/z](https://www.0crat.com/u/borysfan) resigned from [#1416](https://github.com/yegor256/cactoos/issues/1416), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0pdd** [comment]
The puzzle `1411-445dea43` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1453](https://github.com/yegor256/cactoos/issues/1453) is still not solved.

**Comment by 0crat** [comment]
Job was finished in 45 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@eb04ed37-8f5d-4d09-a3fe-0a9e48b1f72b, thread: PQ-C63314D6Z
com.amazonaws.http.AmazonHttpClient$RequestExecutor[1359] com.amazonaws.ResetException: Failed to reset the request input stream;  If the request involves an input stream, the maximum stream buffer size can be configured via request.getRequestClientOptions().setReadLimit(int)
java.io.BufferedInputStream[170] java.io.IOException: Stream closed
```

[0.54.5](https://github.com/zerocracy/farm): CID: [e26fdd1d-e11a-40a6-ac16-9379bd9d1508](https://www.0crat.com/footprint/C63314D6Z/e26fdd1d-e11a-40a6-ac16-9379bd9d1508), Type: "Award points were added"

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1586](https://github.com/yegor256/cactoos/issues/1586) is still not solved; solved: [#1453](https://github.com/yegor256/cactoos/issues/1453).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1714](https://github.com/yegor256/cactoos/issues/1714) is still not solved; solved: [#1453](https://github.com/yegor256/cactoos/issues/1453), [#1586](https://github.com/yegor256/cactoos/issues/1586).
